src/runner.py

```diff
@@ -115,50 +131,56 @@ def create(
         image: str,
         resources: VirtualMachineResources,
         binary: Path,
-        reconcile_interval: int,
         registration_token: str,
     ):
+        """Create the runner instance on LXD and register it on GitHub.
+
+        Args:
+            image: Name of the image to launch the LXD instance with.
+            resources: Resource setting for the LXD instance.
+            binary: Path to the runner binary.
+            registration_token: Token to register the runner on GitHub.
+
+        Raises:
+            RunnerCreateError: _description_
+        """
         logger.info("Creating runner: %s", self.name)
 
         self.instance = self._create_instance(image, resources)
 
         try:
-            self._start_instance(reconcile_interval)
+            self._start_instance()
+            # Wait some initial time for the instance to boot up
+            time.sleep(60)
```

Comment:
Is there a way to check whether it is done rather than wait a set amount of time? That way it would be possible to continue earlier?

Reply:
The next method called `_wait_boot_up` checks every 30 seconds if the instance completes the boot up.
This is just the initial wait prior to the first check.

Reply:
For my understanding, what is the reason for needing the initial wait period if the following command already has a variable wait limit in it?

Reply:
The only reason is due to the first check is pretty much an automatic failure, since virtual machine cannot boot up that fast.